specifyweb/frontend/js_src/lib/components/WbPlanView/mappingReducer.ts

```diff
@@ -251,12 +255,21 @@ export const reducer = generateReducer<MappingState, MappingActions>({
     )
       return state;
 
+    const newLine = replaceItem(state.lines, focusedLine, {
+      ...state.lines[focusedLine],
+      mappingPath: mappingViewMappingPath,
+    });
+
+    const newHeader = generateMappingPathPreview(
+      baseTableName as keyof Tables,
```

Comment:
you should be able to get the base table name from `state`

Reply:
not sure, how would you? 
export type MappingState = State<
  'MappingState',
  {
    readonly showMappingView: boolean;
    readonly showHiddenFields: boolean;
    readonly mappingView: MappingPath;
    readonly mappingsAreValidated: boolean;
    readonly lines: RA<MappingLine>;
    readonly focusedLine: number;
    readonly changesMade: boolean;
    readonly mustMatchPreferences: IR<boolean>;
    readonly autoMapperSuggestions?: RA<AutoMapperSuggestion>;
    readonly openSelectElement?: SelectElementPosition;
    readonly validationResults: RA<MappingPath>;
  }
>;

Reply:
> not sure, how would you?

you are right!
sorry, I forgot that it is coming as a prop, and not part of the state